src/main/java/seedu/duke/lists/RoutineList.java

```diff
@@ -0,0 +1,10 @@
+package seedu.duke.lists;
+
+import java.util.ArrayList;
+
+/**
+ * Adding, deleting and other operations related to work out routines will be executed from here
+ */
+public class RoutineList {
+    public static ArrayList<RoutineList> routineList = new ArrayList<>();
```

Comment:
Shouldn't this be ArrayList<ExerciseList>? As in a list of workouts

Since we're going with the structure of Workouts as a list of exercises, and then a level above being a list of workouts, I think it might be less confusing if the exerciseList is renamed Workout, and then the RoutineList renamed WorkoutList.

Reply:
Got it! Will rename it, thanks